### PR TITLE
📝 Fix type-o and add missing word.

### PR DESCRIPTION
**Pull request by rawatdev**
fix typo in CONTRIBUTUING.md (wil to will)
add missing word in common.json (missing word is milliseconds)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -112,7 +112,7 @@ When you're finished with the changes, create a pull request, also known as a PR
 - Fill the "Ready for review" template so that we can review your PR. This template helps reviewers understand your changes as well as the purpose of your pull request. 
 - Don't forget to [link PR to issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) if you are solving one.
 - Enable the checkbox to [allow maintainer edits](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) so the branch can be updated for a merge.
-Once you submit your PR, your proposal wil be review. We may ask questions or request for additional information.
+Once you submit your PR, your proposal will be review. We may ask questions or request for additional information.
```
Comment (SupremeTechnopriest) [inline CONTRIBUTING.md:115]: Can you also change `review` to `reviewed`

Comment (SupremeTechnopriest) [inline CONTRIBUTING.md:115]: There is another file called `contributing.mdx`. Can you also make these changes there?

`/docs/pages/docs/about/contributing.mdx`

**Review by SupremeTechnopriest — CHANGES_REQUESTED** [review]
Thanks for the PR!  Just a few notes and I will merge in.

File: CONTRIBUTING.md (hunk 1 above)
Comment (rawatdev) [reply CONTRIBUTING.md:115]: ok, I am on it

Comment (rawatdev) [reply CONTRIBUTING.md]: Done

**Review by SupremeTechnopriest — APPROVED** [review]
Looks good! Thanks!

**Comment by SupremeTechnopriest** [comment]
Normally I would require the commit message to be a sentence with punctuation and a capitalized first letter, but ill let it slide this time.

**Pull request merged**